docs/user-docs/build_linux.md

```diff
@@ -0,0 +1,172 @@
+Building and installing Linux
```

Comment:
CI will [require `#` style heading](https://github.com/TrenchBoot/TrenchBoot.github.io/pull/37/commits/439ed82117ffdbfe95feeb7f968fda488a9eabaa#diff-39aecbc26791dd03b9a4fdc1f856769ea73308294b7111d70f279ac9bdef35bdR16-R21), may as well be changed now.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/924f8ee61f342ac95d46d1796f2eb6cd6641726f..eb02c0ec4b2b37327f4ab540d3e3821efabc96d5#diff-9c4b0ba59b4b4f1482abbfcafdbc0e3c091c47cbda592599f4bd03990ee970d3R1